winlogbeat/eventlog/eventlog.go

```diff
@@ -82,30 +82,29 @@ type Record struct {
 func (e Record) ToEvent() beat.Event {
 	m := common.MapStr{
 		"type":          e.API,
-		"log_name":      e.Channel,
-		"source_name":   e.Provider.Name,
-		"computer_name": e.Computer,
 		"record_number": strconv.FormatUint(e.RecordID, 10),
-		"event_id":      e.EventIdentifier.ID,
 	}
+	m.Put("event.id", e.EventIdentifier.ID)
```

Comment:
This is not the same as ECS event.id (at least in my understanding). This more like an identifier for the log message in the application. For example 1102 is "The audit log was cleared".

Reply:
`record_number` is closer to `event.id` but it's only unique given some additional constraints. If we wanted to populate `event.id` I'd do a fast hash of @timestamp + computer_name + log_name + record_number.